src/main/frontend/src/components/Header.js

```diff
@@ -0,0 +1,43 @@
+import { Col, Row } from "react-bootstrap";
+import React from "react";
+import classes from "./Header.module.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faGooglePlus,
+  faFacebook,
+  faTwitter,
+  faInstagram,
+} from "@fortawesome/free-brands-svg-icons";
+
+function Header() {
+  return (
+    <div className={classes.headercontainer}>
+      <Row className={classes.rowfit}>
+        <Col className={classes.socialmediafit}>
+          <ul className={classes.socialmedia}>
+            <li>
+              <FontAwesomeIcon icon={faFacebook} size="2x" />
+            </li>
+            <li>
+              <FontAwesomeIcon icon={faInstagram} size="2x" />
+            </li>
+            <li>
+              <FontAwesomeIcon icon={faTwitter} size="2x" />
+            </li>
+            <li>
+              <FontAwesomeIcon icon={faGooglePlus} size="2x" />
+            </li>
```

Comment:
You can extract this to separate component and reuse here and in footer. Also it would be easier to make it as 
```suggestion
            <div>
              <FontAwesomeIcon icon={faFacebook} size="2x" />
              <FontAwesomeIcon icon={faInstagram} size="2x" />
              <FontAwesomeIcon icon={faTwitter} size="2x" />
              <FontAwesomeIcon icon={faGooglePlus} size="2x" />
            </div>
```
and have class for div: max-width: 150px display: flex, justify-content: space-between